packages/app/src/components/actions/deposit.tsx

```diff
@@ -90,13 +95,13 @@ const Deposit: FC<Props> = ({ name, symbol, logo, tokenBalance, tenderTokenBalan
               <FormField
                 fill
                 label="Deposit Amount"
-                name="depositAmount"
+                protocolName="depositAmount"
```

Comment:
of course I did not catch this before earlier, this broke input validation @kyriediculous